packages/core/src/event-store/migration.test.ts

```diff
@@ -0,0 +1,186 @@
+import "@/utils/globals";
+
+import { hexToNumber, RpcBlock, RpcTransaction } from "viem";
+import { beforeEach, expect, test } from "vitest";
+
+import { setupEventStore } from "@/_test/setup";
+
+beforeEach(
+  async (context) => await setupEventStore(context, { skipMigrateUp: true })
+);
+test("setup creates tables", async (context) => {
```

Comment:
Is there a reason you're including this test here? It's already in `store.test.ts`.

Reply:
yeah this doesn't need to be here, removing this